pkg/types/kline.go

```diff
@@ -508,3 +508,96 @@ func (k KLineWindow) SlackAttachment() slack.Attachment {
 }
 
 type KLineCallback func(kline KLine)
+
+type KValueType int
+
+const (
+	kOpenValue KValueType = iota
```

Comment:
if the field is not assigned, the default value will be `kOpenValue` (since it's 0) is that what you want?

Reply:
fixed